custom_components/echonetlite/fan.py

```diff
@@ -8,15 +8,18 @@
     PRECISION_WHOLE,
 )
 from . import get_device_name
-from .const import CONF_FORCE_POLLING, DATA_STATE_ON, DOMAIN
+from .const import (
+    CONF_FORCE_POLLING,
+    DATA_STATE_ON,
+    DOMAIN,
+    ENL_FANSPEED,
```

Comment:
following the import ref on this one, its coming from pychonet.HomeAirConditioner https://github.com/scottyphillips/echonetlite_homeassistant/blob/8fb38203bcdd010b5fca5dc99ac5c7337323b258/custom_components/echonetlite/const.py#L34C5-L34C17 which I'm guessing while it works and the value is the same its not 100% correct